src/main/java/seedu/address/storage/JsonUserPrefsStorage.java

```diff
@@ -32,8 +32,9 @@ public Optional<UserPrefs> readUserPrefs() throws DataConversionException {
 
     /**
      * Similar to {@link #readUserPrefs()}
+     *
      * @param prefsFilePath location of the data. Cannot be null.
-     * @throws DataConversionException if the file format is not as expected.
+     * @throws DataConversionException if the loading of the preference file fails.
```

Comment:
This is phrasing is slightly different from the one used in `readAddressBook`. Don't vary things unless there is a good reason.